src/extension.ts

```diff
@@ -26,19 +32,67 @@ export function activate(context: ExtensionContext) {
       // Register the server for java
       documentSelector: ['xml'],
       revealOutputChannelOn: RevealOutputChannelOn.Never,
-      initializationOptions: {"lsp4xml.logPath": logfile}
-      
+      initializationOptions: { "lsp4xml.logPath": logfile },
```

Comment:
all the settings should be sent in the initialization options, else logs.client is ignored on startup